Service/Model.SqlCE/TodoModel.Context.cs

```diff
@@ -0,0 +1,31 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//    This code was generated from a template.
+//
+//    Manual changes to this file may cause unexpected behavior in your application.
+//    Manual changes to this file will be overwritten if the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Model.SqlCe
+{
+    using System;
+    using System.Data.Entity;
+    using System.Data.Entity.Infrastructure;
+    
+    public partial class TodoEntities : DbContext
```

Comment:
TodoEntities means a collection of Todo objects. Consider changing name, something like TodoDbContext